package.json

```diff
@@ -46,7 +46,7 @@
   },
   "devDependencies": {
     "@types/qunit": "^2.0.31",
-    "amd-name-resolver": "^1.0.0",
+    "amd-name-resolver": "^1.3.1",
```

Comment:
Do we need to update this?

Reply:
On a fresh checkout, running yarn produces the error below. This was how Tom and I resolved it. Is it not correct?

`mpascual@mpascual-mn2.linkedin.biz ~/P/tmp> git clone git@github.com:glimmerjs/glimmer-vm.git
Cloning into 'glimmer-vm'...
remote: Enumerating objects: 38, done.
remote: Counting objects: 100% (38/38), done.
remote: Compressing objects: 100% (37/37), done.
remote: Total 43688 (delta 0), reused 14 (delta 0), pack-reused 43650
Receiving objects: 100% (43688/43688), 10.59 MiB | 7.32 MiB/s, done.
Resolving deltas: 100% (28556/28556), done.
mpascual@mpascual-mn2.linkedin.biz ~/P/tmp> cd glimmer-vm
mpascual@mpascual-mn2.linkedin.biz ~/P/t/glimmer-vm (master)> yarn
yarn install v1.17.3
[1/4] 🔍  Resolving packages...
warning Resolution field "typescript@3.3.3" is incompatible with requested version "typescript@~3.2.1"
[2/4] 🚚  Fetching packages...
error https://github.com/ember-cli/amd-name-resolver.git: Extracting tar content of undefined failed, the file appears to be corrupt: "Invalid tar header. Maybe the tar is corrupted or it needs to be gunzipped?"
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
mpascual@mpascual-mn2.linkedin.biz ~/P/t/glimmer-vm (master)>                                                                                                                                                  1 ↵
`